automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl

```diff
@@ -261,6 +261,15 @@ groups:
       summary: "{{ $labels.testnet }} block production is critically low (there has been less than 1 block in the last hour)"
       description: "{{ $value }} blocks have been produced on network {{ $labels.testnet }} in the last hour (according to some node)."
 
+  - alert: DisconnectedBlocksPerHour
+    expr: max by (testnet) (increase(Coda_Rejected_blocks_no_common_ancestor ${rule_filter} [${alert_timeframe}])) > 10
+    labels:
```

Comment:
These may come from a single block producer, so probably lower the threshold? How about warning when there is at least 1 and then critical alert if >5? 
@nholland94 do you have thoughts on this?

Reply:
I agree that we will want some warnings for anything > 0. I don't have a super great sense of a reasonable threshold for an alert would be, mostly depends on what `alert_timeframe` is. If it's 1 hour, for instance, I would say anything > 3 may be worth an alert. We want to quickly act if a significant amount of stake is contributing to a side fork.